src/BRSRC13/CORE/FW/register.c

```diff
@@ -156,17 +186,17 @@ int BrRegistryEnum(br_registry* reg, char* pattern, br_enum_cbfn* callback, void
 // IDA: void* __usercall BrRegistryNewStatic@<EAX>(br_registry *reg@<EAX>, br_registry_entry *base@<EDX>, int limit@<EBX>)
 void* BrRegistryNewStatic(br_registry* reg, br_registry_entry* base, int limit) {
     LOG_TRACE("(%p, %p, %d)", reg, base, limit);
-    NOT_IMPLEMENTED();
+    return NULL;
```

Comment:
whats the reasoning behind this? Either its fully implemented, or not. Returning NULL could lead to unexpected behavior that takes us longer to debug?

Reply:
The implementations were empty, simply returning NULL.
Perhaps these are functions that can conditionally contain code with some compiler option.

If you desire, I can revert this change.

Reply:
ok, all good.